content/en/docs/concepts/storage/storage-limits.md

```diff
@@ -64,15 +64,14 @@ type and enforces the appropriate maximum number of volumes for the node. For ex
 
 * On
 <a href="https://cloud.google.com/compute/">Google Compute Engine</a>,
-up to 127 volumes can be attached to a node, [depending on the node
+up to 127 volumes can be attached to a Node, [depending on the Node
 type](https://cloud.google.com/compute/docs/disks/#pdnumberlimits).
 
-* For Amazon EBS disks on M5,C5,R5,T3 and Z1D instance types, Kubernetes allows only 25
-volumes to be attached to a Node. For other instance types on
+* On
 <a href="https://aws.amazon.com/ec2/">Amazon Elastic Compute Cloud (EC2)</a>,
-Kubernetes allows 39 volumes to be attached to a Node.
+up to 127 volumes can be attached to a Node, depending on the Node type. For more details, refer to [Amazon EBS volume limits for Amazon EC2 instances](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/volume_limits.html).
```

Comment:
In the same [page](https://kubernetes.io/docs/concepts/storage/storage-limits/#kubernetes-default-limits), does the 'Maximum volumes per Node' value in the table for 'Amazon Elastic Block Store (EBS)' (around [Line 33 (here)](https://github.com/kubernetes/website/blob/f346358db28126431f9b1da8f8fde0b53c5ff1f9/content/en/docs/concepts/storage/storage-limits.md?plain=1#L33) in the earlier section) also need to be updated to reflect these changes?

Reply:
I was worried that those values are for old in-tree cloud persistent volume types, which is why the Azure and GKE bullet points also don't align with those in-table values. Not positive though.

IMO eventually it would be best to eliminate that table.

Reply:
I'd remove that column, at the very least.